src/test/fuzz/connman.cpp

```diff
@@ -160,6 +160,15 @@ FUZZ_TARGET(connman, .init = initialize_connman)
                     /*strDest=*/fuzzed_data_provider.ConsumeBool() ? nullptr : random_string.c_str(),
                     /*conn_type=*/conn_type,
                     /*use_v2transport=*/fuzzed_data_provider.ConsumeBool());
+            },
+            [&] {
+                connman.SetNetworkActive(true);
```

Comment:
In b1b74a13adb8e943d90fbe56c4a706b0ae91335a: nit: I don't think to set network active here is a must, we could fuzz it with both network active and inactive.

Reply:
Right, done!